src/clj_fdb/core.clj

```diff
@@ -120,3 +123,110 @@
   [^TransactionContext tc ^Range rg]
   (let [tr-fn (fn [^Transaction tr] (ftr/clear-range tr rg))]
     (ftr/run tc tr-fn)))
+
+(defn set-subspaced-key
+  "Takes the following:
+  - TransactionContext `tc`
+  - Subspace `s` which will be used to namespace the key
+  - Value to be stored `v`
+
+  and stores `v` against key constructed using `s` and `t` in DB. Returns nil.
+
+  Optionally, you can pass in `:t` and `:valfn` as follows:
+
+  - `:t` will be used along with `s` to construct key. `t` should of type `Tuple` class.
+  If `t` is not passed, empty `Tuple` will be used.
```

Comment:
Make `t` a compulsory argument. If the user wants to set a value against the empty tuple, he can pass the tuple in himself.